lib/rdf2marc/models/subject_access_field/event_name.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Rdf2marc
+  module Models
+    module SubjectAccessField
+      # Model for 647 - Named Event.
+      # See https://www.loc.gov/marc/bibliographic/bd647.html
+      class EventName < Struct
+        attribute :thesaurus, Types::String.default('not_specified').enum('lcsh',
+                                                                          'lcsh_childrens_literature',
+                                                                          'mesh',
+                                                                          'nal_subject_authority',
+                                                                          'not_specified',
+                                                                          'canadian_subject_headings',
+                                                                          'répertoire_de_vedettes-matière',
+                                                                          'subfield2')
```

Comment:
is this enum repeated in a bunch of places?  Should that somehow be a part of the Thesaurus class you just created?

Reply:
Good idea. I added it to the `Types` class.